swagger.json

```diff
@@ -0,0 +1,11140 @@
+{
```

Comment:
I don't think we need to commit this file

Reply:
Douh!